paprika/build.py

```diff
@@ -74,6 +101,39 @@ def countresidues(filename='tleap.in', directory='.', choice='water_residues',
     if p.poll() is None:
         p.kill()
 
+    return output
+
+
+def basic_tleap(tleapfile, directory='.', pdbfile=None, saveprefix=None):
```

Comment:
Better name than `basic`? And say explicitly how it is different than `run_tleap`. Is one of them a wrapper?

Reply:
yeah, I'm open to new names.  
basic: reads user's template tleap file, writes the apr version (with new pdb, for example), and then runs tleap.
run_tleap: just the execution of tleap step.  I gave it its own function to avoid redundant code.

Reply:
Hmm. Looking at lines 116-123 below, I'm confused a little. The existing `tleap` file is read into `tleaplines` and this list of strings is passed to the `write_tleapin` which should write a new input file. Where do we replace the new PDB, for example? Is it during the reading of `tleaplines` from the user file? I guess it would make things easier to debug/understand if we somehow clarified what's changing from reading the user file → writing of the new file. I don't have a great idea for that off the top of my head.

Reply:
Currently `read_tleaplines` allows you to swap out the existing `loadpdb xxx.pdb` for an alternate `loadpdb yyy.pdb`.  Maybe we should move that functionality to `write_tleapin`.

Reply:
Yes, can you add to #18?